mappings/net/minecraft/entity/Positioned.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_ogzwtxja net/minecraft/entity/Positioned
+	METHOD m_deqgaqvk of (Lnet/minecraft/unmapped/C_vgpupfxx;Lnet/minecraft/unmapped/C_xpuuihxf;Lnet/minecraft/unmapped/C_cdctfzbn;)Lnet/minecraft/unmapped/C_ogzwtxja;
+	METHOD m_iaptyxom asLivingEntity ()Lnet/minecraft/unmapped/C_usxaxydn;
+	METHOD m_kqxvnqqn world ()Lnet/minecraft/unmapped/C_cdctfzbn;
+	METHOD m_rxhilfnh visualYawDegrees ()F
+	METHOD m_wmxlrpiw pos ()Lnet/minecraft/unmapped/C_vgpupfxx;
+	CLASS C_kkvdsmey Simple
+		FIELD f_ecogiffv visualYawDegrees F
+		FIELD f_mbhyquak world Lnet/minecraft/unmapped/C_cdctfzbn;
+		FIELD f_mkdobvlt pos Lnet/minecraft/unmapped/C_vgpupfxx;
```

Comment:
these record components are the reason `pos()` and `world()` lack the `get` prefix
this affects `Tamable` and `StorageVehicle`